book/header.css

```diff
@@ -341,6 +356,39 @@
 .sub-nav__item--anchor--profile p {
   margin: 0;
 }
+.sub-nav__item--organizations {
+  padding-left: 0;
+  padding-right: 0;
+}
+.sub-nav__item--organizations__container {
+  width: 200px;
+}
+.sub-nav__item__organization {
```

Comment:
@jennyknuth, we can probably remove organization classes here too just to keep things tidy. Adds an extra step to the portal -> gitbook styling but nice to have

Reply:
Good point and it helped me reconsider. To ease maintenance, the strategy now is to use the css from the portal as the source of truth and copy it verbatim. True, much of it is unused, but it simplifies updates so is simpler in its own way.